upgrade-2.x.sh

```diff
@@ -760,7 +760,7 @@ function get_delta_token() {
 #######################################
 function find_delta() {
     # we need to strip the target_version's variant tag to query the API properly
```

Comment:
- Should this comment be removed, or possibly moved up to `get_image_location()`?

- Out of curiosity, what prompted the change?

Reply:
good catch! I moved it since i call that same function a second time in line 1176, and it's a transformation that more precisely should be done there anyway since that's where it's used (in `get_image_location()`).